### PR TITLE
[FEATURE] item table improvements

### PR DESCRIPTION
**Pull request by turbo-ele**
**Tasks:**
- [x] done
- [x] todo
- [x] got feedback

**Informations:**

| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| Tests pass? | yes |
| Fixed tickets | none |
| Doc | none |

### CODE REVIEW

File: Resources/public/js/components/item-table/main.js [hunk 1]
```diff
@@ -523,7 +544,7 @@ define([
             this.sandbox.dom.remove($row);
 
             // decrease row counter
-            this.rowCount--;
+//            this.rowCount--;
```
Comment (christianbader) [inline Resources/public/js/components/item-table/main.js]: hm?

File: Resources/public/js/components/item-table/main.js [hunk 2]
```diff
@@ -712,6 +733,42 @@ define([
...
+                    name: 'overlay@husky',
+                    options: {
+                        el: $overlay,
+                        title: this.sandbox.translate('test 123'),
+                        openOnStart: true,
+                        removeOnClose: false,
+                        skin: 'wide',
+                        data: $content,
+                        okCallback: function() {
+
```
Comment (christianbader) [inline Resources/public/js/components/item-table/main.js:761]: ?

**Comment by christianbader** [comment]
Ack

File: Resources/public/js/components/item-table/main.js (hunk 2 above)
Comment (turbo-ele) [inline Resources/public/js/components/item-table/main.js:761]: will be needed, when settings overlay is implemented (ongoing process)

**Pull request merged**